{
  "Severity": "HIGH",
  "UpdatedAt": "2024-08-27T19:50:55Z",
  "Package": {
    "Ecosystem": "PIP",
    "Name": "taipy"
  },
  "Advisory": {
    "DatabaseId": 257957,
    "Id": "GSA_kwCzR0hTQS1wcDg0LXYzbXctZ2c0d84AA--l",
    "GhsaId": "GHSA-pp84-v3mw-gg4w",
    "References": [
      {
        "Url": "https://github.com/Avaiga/taipy/security/advisories/GHSA-pp84-v3mw-gg4w"
      },
      {
        "Url": "https://github.com/advisories/GHSA-pp84-v3mw-gg4w"
      }
    ],
    "Identifiers": [
      {
        "Type": "GHSA",
        "Value": "GHSA-pp84-v3mw-gg4w"
      }
    ],
    "Description": "### Summary\nCVEs on latest 3.1.1\n\n### Details\n## SECURITY ISSUES\n### CVE-2024-1681: flask-core \u003c4.0.1 \nlatest version of taipi 3.1.1 needs \u003c=4.0.0 \n\n### CVE-2024-5629: pymongo \u003c4.6.3 \n#latest version of taipi 3.1.1 needs \u003c=4.6.1 \n\n### PoC\nplease upgrade to these versions \n\n### Impact\npre-commit breaks\n",
    "Origin": "UNSPECIFIED",
    "PublishedAt": "2024-08-27T19:50:54Z",
    "Severity": "HIGH",
    "Summary": "Taipy 3.1.1 affected by CVEs on flask-core and pymongo",
    "UpdatedAt": "2024-08-27T19:50:55Z",
    "WithdrawnAt": "",
    "CVSS": {
      "Score": 0,
      "VectorString": ""
    }
  },
  "Versions": [
    {
      "FirstPatchedVersion": {
        "Identifier": ""
      },
      "VulnerableVersionRange": "\u003c= 3.1.1"
    }
  ]
}